agasc/agasc.py

```diff
@@ -12,6 +12,8 @@
 
 DATA_ROOT = ska_path('data', 'agasc')
 
+tables_open_file = getattr(tables, 'open_file', None) or tables.openFile
```

Comment:
This is a PyTables 2.x vs PyTables 3.x issue.  They made a major API change with 3.x.  I was thinking about a more general compatibility layer, but there are few enough instances of hdf5 access in our production code that it probably is not worth it.